docs/quickstart.md

```diff
@@ -45,42 +45,39 @@ You should get an output similar to this:
 🚢 FuseML installing...
 
 Configuration...
-  🧭  system_domain: 
+  🧭  system_domain:
+  🧭  extension_repository: https://raw.githubusercontent.com/fuseml/extensions/main/installer/
```

Comment:
OK, this should not point to `main` in released versions...

Reply:
... but that's OK for `main` branch of docu repo. We just need to fix it when branching